app/controllers/training_requests_controller.rb

```diff
@@ -0,0 +1,54 @@
+class TrainingRequestsController < ApplicationController
+  admin_tab :index
+
+  before_filter :authenticate_user!
+  before_filter :check_acting_as
+
+  load_and_authorize_resource
+
+  layout "two_column"
```

Comment:
I'm not sure this should be `two_column`. The alignment looks a little funky at wider windows.